app/web/features/communities/events/EventPage.tsx

```diff
@@ -51,7 +51,9 @@ export const useEventPageStyles = makeStyles<Theme, { eventImageSrc: string }>(
       },
       width: "100%",
       objectFit: ({ eventImageSrc }) =>
-        eventImageSrc === eventImagePlaceholder ? "contain" : "cover",
+        eventImageSrc === "/img/eventImagePlaceholder.svg"
```

Comment:
```suggestion
        eventImageSrc === eventImagePlaceholderUrl
```
Could use that from `appConstants`?